schemas/blocks/interstitial.json

```diff
@@ -10,6 +10,10 @@
       "number": {
         "$ref": "https://eq.ons.gov.uk/common_definitions.json#/non_empty_string"
       },
+      "page_title": {
+        "$ref": "https://eq.ons.gov.uk/common_definitions.json#/non_empty_string",
+        "description": "Custom page title used on the interstitial page."
+      },
```

Comment:
I think it would be better to make all uses of `page_title` use a reference with a generic description. The context of what page the property is used on is covered by it being in that block types definition.